env.world

```diff
@@ -14,4 +14,4 @@ SAVEINTERVAL=1800
 BACKUPS=4
 BACKUPSHORT=7200
 BACKUPLONG=43200
-CROSSPLAY=TRUE
```

Comment:
crossplay on FALSE for default settings, because it was an issue for me that I had to investigate in the first place.

Reply:
Yes, I read something about the cross play on Linux that it does not work. 
In the test done in my server it works fine, but I don't understand why other people has that problems, some of the dll are not loaded correctly on the startup and block the server.
But it's ok, I would accept the PR but better, set CROSSPLAY=FALSE 
This is the example values, so I prefer to be filled that other people can see all the variables that can be used.

Reply:
I did change the value from TRUE to FALSE